group_vars/workstation.yml

```diff
@@ -1,3 +1,16 @@
 ---
-LWG_theme_name: cv
 workstation_mount: true
+greeter_theme:
+  accent1: '#4dd0e1'
+  accent2: '#00796b'
+  accent2_hover: '#009688'
+  default_session: 'cinnamon'
+  username_entry: 'fill'
+  wallpaper: 'https://verdict4u.files.wordpress.com/2016/09/google-now-wallpaper-1.png'
+  session_api: 'http://ldap-api.collegiumv.org/%s/desktopEnvironment/0'
+  aup: |
+    [1] Unauthorized use is prohibited;
+    [2] Usage may be subject to security testing and monitoring;
+    [3] Misuse is subject to criminal prosecution; and
+    [4] Users have no expectation of privacy except as otherwise provided by applicable privacy laws.
```

Comment:
Line too long 😞 

Could we increase the limit to 160 characters?

Reply:
I'm fine with an increase.